course_catalog/views_userlist_test.py

```diff
@@ -0,0 +1,314 @@
+"""Tests for course_catalog UserList views"""
```

Comment:
Nothing in this file is new, it was just moved out of `views_test.py` because it had gotten too big (pylint was complaining about >1000 lines).